app/src/main/java/org/dicekeys/app/RecipeBuilder.kt

```diff
@@ -0,0 +1,67 @@
+package org.dicekeys.app
+
+import org.dicekeys.api.DerivationRecipe
+import org.dicekeys.crypto.seeded.DerivationOptions
+import java.net.URL
+
+/*
+ * A builder to create custom or template based Recipes
+ */
+class RecipeBuilder(val template: DerivationRecipe?) {
+    private var domainList: List<String> = listOf()
+    private var sequence = 1
+    private var lengthInChars = 0
+
+    fun reset(){
+        domainList = listOf()
+        sequence = 1
+        lengthInChars = 0
+    }
+
+    fun updateDomains(domains: String){
+
+        // Check if is valid URL
+        try{
+            val url = URL(domains)
+            domainList = listOf(url.host)
+            return
+        }catch (e: Exception){
+            e.printStackTrace()
```

Comment:
Was this just for debugging?

Reply:
It's used only for the test, no need to reset it on the app because always we set the default values.